src/config/other.ts

```diff
@@ -93,6 +93,7 @@ export const otherRules = {
   "promise/valid-params": "error",
 
   "react-compiler/react-compiler": "error",
+  "react-server-components/use-client": "error",
```

Comment:
Added as an error, but then I'm thinking about a overriding this on `platform` for warn, and open PRs to fix where it breaks and remove those afterwards. These are the packages that broke line:

```ts
- nx run pika:lint
- nx run howl-run-environment-utils:lint
- nx run recorder:lint
- nx run playground-playwright:lint
- nx run keyboard:lint
- nx run react-state-machine:lint
- nx run tester-sandbox:lint
- nx run wolfui:lint
- nx run app:lint
```